debezium-testing/debezium-testing-testcontainers/src/test/java/io/debezium/testing/testcontainers/ApicurioRegistryTest.java

```diff
@@ -55,11 +55,10 @@
     private static final String DEBEZIUM_VERSION = "1.3.0.Final";
     private static final String APICURIO_VERSION = "1.3.2.Final";
 
-    private static final DockerImageName POSTGRES_DOCKER_IMAGE_NAME = DockerImageName.parse("debezium/postgres:11")
-            .asCompatibleSubstituteFor("postgres");
-
     private static Network network = Network.newNetwork();
 
+    private static final PostgresInfrastructure POSTGRES_INFRASTRUCTURE = PostgresInfrastructure.getDebeziumPostgresDefaultInfrastructure(network);
+
     private static GenericContainer<?> apicurioContainer = new GenericContainer<>("apicurio/apicurio-registry-mem:" + APICURIO_VERSION)
```

Comment:
@ani-sha Could you please extend the scope of this PR and sove it also for other container like `apicurio` here or `connect` below? Ideally all images used in all tests should be defined in a single place.

Reply:
@jpechane ok sure, on it.

Reply:
Do I sense scope creep here ;) But agreed, it's a reasonable expansion.